src/kOS/Binding/SteeringManager.cs

```diff
@@ -1391,7 +1361,7 @@ public void WriteCSVs()
             }));
         }
 
-        public void RemoveInstance(SharedObjects shared)
```

Comment:
we have a static and instance RemoveInstance that take different arguments? I hope i was successful in untangling them.